src/applications/personalization/profile/components/personal-information/ContactInformationField.jsx

```diff
@@ -327,7 +315,33 @@ class ContactInformationField extends React.Component {
       content = (
         <EditView
           refreshTransaction={this.refreshTransaction}
-          {...childProps}
+          analyticsSectionName={this.props.analyticsSectionName}
+          apiRoute={this.props.apiRoute}
+          blockEditMode={this.props.blockEditMode}
+          clearErrors={this.clearErrors}
+          clearTransactionRequest={this.props.clearTransactionRequest}
+          convertCleanDataToPayLoad={this.props.convertCleanDataToPayLoad}
+          createTransaction={this.props.createTransaction}
+          deleteDisabled={this.props.deleteDisabled}
+          data={this.props.data}
+          field={this.props.field}
+          fieldName={this.props.fieldName}
+          formSchema={this.props.formSchema}
+          hasUnsavedEdits={this.props.hasUnsavedEdits}
+          isEmpty={this.props.isEmpty}
+          onCancel={this.onCancel}
+          onChangeFormDataAndSchemas={this.onChangeFormDataAndSchemas}
+          onDelete={this.onDelete}
+          title={this.props.title}
+          onEdit={this.onEdit}
+          onSubmit={this.onSubmit}
+          showEditView={this.props.showEditView}
+          transaction={this.props.transaction}
+          transactionRequest={this.props.transactionRequest}
+          uiSchema={this.props.uiSchema}
+          validateAddress={this.props.validateAddress}
+          EditView={this.props.EditView}
```

Comment:
Was going to make some comments here then I saw this in your PR message:
> This PR is not yet responsible for weeding out unused props in the address components, and a follow-up PR will also be created to minimize the number of props passed down the component chain even further.

so, 👍